lightning-invoice/src/lib.rs

```diff
@@ -480,7 +490,7 @@ pub struct PayeePubKey(pub PublicKey);
 /// Positive duration that defines when (relatively to the timestamp) in the future the invoice
 /// expires
 #[derive(Clone, Debug, Hash, Eq, PartialEq, Ord, PartialOrd)]
-pub struct ExpiryTime(Duration);
+pub struct ExpiryTime(pub Duration);
```

Comment:
Shouldn't `pub` be removed from here too, as per Matt's comment? https://github.com/lightningdevkit/rust-lightning/pull/2730#discussion_r1393061598

```suggestion
pub struct ExpiryTime(Duration);
```